net/dnscache/dnscache.go

```diff
@@ -25,6 +25,8 @@ import (
 	"tailscale.com/util/singleflight"
 )
 
+var zaddr netip.Addr
```

Comment:
oh i meant as a var in the func, global var seems overkill?

Reply:
oh i see it is used in multiple funcs

Reply:
Yeah, there's a bunch of instances of it, so it seemed better to just make one global declaration.